templates/elected_representative/list.html.twig

```diff
@@ -105,3 +125,54 @@
         </section>
     </main>
 {% endblock %}
+
+{% block final_javascripts %}
+    {{ parent() }}
+
+    {% import 'javascript.js.twig' as js %}
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            $('td.labels').on('click', '[id^="show_more_labels_"], [id^="show_less_labels_"]', function() {
+                let elementId = $(this).attr('id');
+                $(this).click(function(){
+                    $('#'+elementId).toggleClass('visually-hidden');
+                    if (elementId.includes('more')) {
+                        $('#'+elementId.replace('show_more_labels', 'show_less_labels')).toggleClass('visually-hidden');
+                        $('#'+elementId.replace('show_more_labels', 'other_labels')).toggleClass('visually-hidden');
+                    } else {
+                        $('#'+elementId.replace('show_less_labels', 'show_more_labels')).toggleClass('visually-hidden');
+                        $('#'+elementId.replace('show_less_labels', 'other_labels')).toggleClass('visually-hidden');
+                    }
+                });
+            });
+
+            App.runUserListDefinitions(
+                '{{ constant('App\\Controller\\EnMarche\\UserListDefinition\\ElectedRepresentativeUserListDefinitionController::MEMBER_TYPE') }}',
+                '{{ constant('App\\Entity\\UserListDefinitionEnum::TYPE_ELECTED_REPRESENTATIVE') }}',
+                '.user-list-definition-manager-wrapper',
+                'input[name="items[]"]',
+                '#items-check-all',
+                {{ total_count }},
+                (labels) => {
```

Comment:
`function (labels) {...}` instead of arrow function, arrow function (ES6) should be transpiled into ES5 (doing by babel actually).